content/realtime/encryption.textile

```diff
@@ -52,6 +50,8 @@ h2(#getting-started). Getting started
   channel.publish("example", "message data");
 ```
 
+Note that the @key@ should not be a password, but a cryptographic key - randomly generated, 128 or 256 bits long, binary or base64-encoded. If you wish to encrypt messages with a password (for example, one inputted by a user), you should use a "key derivation function":https://en.wikipedia.org/wiki/Key_derivation_function to transform that into a key. The client libraries are also capable of "generating a random key":#generate-random-key for you.
```

Comment:
I suggest "passphrase" instead of "password".

Instead of just "randomly generated" I think we should say "from a secure random source".

"inputted" - is that a word? Maybe "entered" ?